src/+matlode/+rk/+erk/ERK.m

```diff
@@ -124,7 +107,7 @@
                 hHist(1) = hN;
                 
                 %Subject to change
-                hmin = 64 * eps(tCur);
+                hmin = 64 * eps(tCur) * tdir;
```

Comment:
Basically the same complaint as before. I am not sure about the online changiness of this.